app/src/main/java/com/codesoom/assignment/application/ToyService.java

```diff
@@ -0,0 +1,60 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Toy;
+import com.codesoom.assignment.domain.ToyRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class ToyService {
+    private final ToyRepository toyRepository;
+
+    public ToyService(ToyRepository toyRepository) {
+        this.toyRepository = toyRepository;
+    }
+
+    public List<Toy> getProducts() {
+        return toyRepository.findAll();
+    }
+
+    public Toy getProduct(Long id) {
+        return toyRepository.findById(id)
+                .orElseThrow(() -> new ProductNotFoundException(id));
+    }
+
+    public Toy deleteProduct(Long id) {
+        Toy toy = toyRepository.findById(id)
+                .orElseThrow(() -> new ProductNotFoundException(id));
```

Comment:
에러 메시지를 좀 더 디테일하게 작성해봐도 좋겠죠.
```suggestion
                .orElseThrow(() -> new ProductNotFoundException(String.format("id [%d]에 해당하는 상품을 찾지 못했으므로 삭제 작업을 할 수 없습니다.", id)));
```